herddb-core/src/test/java/herddb/sql/CalcitePlannerTest.java

```diff
@@ -486,6 +486,44 @@ public void explainPlanTest() throws Exception {
         }
     }
 
+    @Test
+    public void explainPlanTestIsNullExpression() throws Exception {
```

Comment:
is this test useful ?

Reply:
In the current state, it is not. I removed it.